sips/sip-03x/sip-03x-btc-addresses-for-stacks-transactions.md

```diff
@@ -0,0 +1,215 @@
+Discussions-To: [https://forum.stacks.org/t/proposing-bitcoin-addresses-for-stacks/17910?u=larry](https://forum.stacks.org/t/proposing-bitcoin-addresses-for-stacks/17910?u=larry)   
+Requires: None   
+Replaces: None   
+Superceded-By: None
+
+## **Abstract**
+
+This SIP proposes the adoption of Bitcoin address formats (P2PKH and P2WPKH) as the standard user-facing address format for Stacks network transactions, including those involving sBTC. Currently, Stacks transactions use a unique address format distinct from that of Bitcoin. Aligning with Bitcoin's established address formats aims to reduce user friction, improve the user experience for Bitcoin users interacting with sBTC and the Stacks layer, and enhance the perception of sBTC as a Layer 2 Bitcoin solution. This change is intended to be implemented at the developer tool, library, and application level without requiring consensus changes to the Stacks blockchain.
+
+## **Copyright**
+
+This SIP is made available under the terms of the BSD-2-Clause license, available at [https://opensource.org/licenses/BSD-2-Clause](https://opensource.org/licenses/BSD-2-Clause). This SIP’s copyright is held by New Internet Labs Limited.
+
+## **Introduction**
+
+sBTC is designed to make Bitcoin programmable and enable faster transactions, positioning it as a significant component in the growth of decentralized finance and payments on Bitcoin. However, the current Stacks-specific address format presents a barrier to adoption for existing Bitcoin users, who are accustomed to Bitcoin's native address formats. Building wallets to handle both stacks addresses and bitcoin addresses currently requires duplication of user interface components and send/receive user flows. The resulting user interfaces can lead to confusion and a less intuitive user experience. Other Layer 2 solutions have faced similar UX challenges. The core problem this SIP addresses is the user experience friction caused by requiring Bitcoin users to use a non-Bitcoin address format when interacting with sBTC.  
+   
+**The proposal is to enable the use of standard Bitcoin addresses (legacy P2PKH and native SegWit P2WPKH) for sending and receiving sBTC and other Stacks-based assets.** This would allow sBTC to look and function more like Bitcoin from a user's perspective, leveraging existing user familiarity and tools, such as bitcoin payment QR codes and payment flows. The goal is to make interacting with sBTC as seamless as interacting with Bitcoin on the base layer, thereby encouraging broader adoption.
+
+## **Specification**
+
+The proposed change involves modifications at the application and library level, not the consensus layer of the Stacks blockchain.
+
+### **Address Display**
+
+User-facing applications (wallets, exchanges, explorers) MUST display Stacks account addresses as a Bitcoin native SegWit (P2WPKH) format by default. Support for display of legacy P2PKH format MAY also be included if necessary, but is generally discouraged.
```

Comment:
Re: use of native segwit, the underlying bytes of a Stacks address today are in 1-to-1 correspondence with a legacy P2PKH or P2SH address.  It is already possible to seamlessly convert between a legacy Bitcoin address and a Stacks address, which would make it easier to recover BTC accidentally sent to a Stacks address (per my above comment).  Why must the representation of a Stacks address be a segwit address?